releases/v20.1.0-alpha.20200210.md

```diff
@@ -0,0 +1,351 @@
+- `cockroach zip` now emits the `goroutine` file in the proper sub-directory when the corresponding call fails with an error. [#44064][#44064]
+- `cockroach debug zip` is again able to operate correctly and continue to iterate over all nodes if one of the nodes does not deliver its goroutine dumps. It would previously prematurely and incorrectly stop with an incomplete dump; this was a regression introduced in v19.2. [#44064][#44064]
+- The file generated by `cockroach gen haproxy` no longer gets an executable bit. The executable bit was previously placed in error. [#44043][#44043]
+- Fixed internal error of the form "x FK cols, only y cols in index" in some cases when inserting into a table with foreign key references. [#44031][#44031]
+- CockroachDB now ensures the import cleanup is only run once. [#43960][#43960]
+- Converted a panic in `golang.org/x/text/language/tags.go` when using collated strings to an error. [#44103][#44103]
+- SQL mutation statements that target tables with no foreign key relationships now correctly read data as per the state of the database when the statement started execution. This is required for compatibility with PostgreSQL and to ensure deterministic behavior when certain operations are parallelized. Prior to this fix, a statement [could incorrectly operate multiple times](https://en.wikipedia.org/wiki/Halloween_Problem) on data that itself was writing, and potentially never terminate. This fix is limited to tables without FK relationships, and for certain operations on tables with FK relationships; in other cases, the fix is not active and the bug is still present. A full fix will be provided in a later release. [#42862][#42862]
+- CockroachDB now properly supports using `--url` with query options (e.g., `application_name`) but without specifying `sslmode`. The default of `sslmode=disable` is assumed in that case. [#44113][#44113]
+- The GC process has been improved to paginate the key versions of a key to fix OOM crashes, which can occur when there are extremely large numbers of versions for a given key. [#43862][#43862]
+- Removed statistics information from backup jobs' payload information to avoid excessive memory utilization when issuing commands such as [`SHOW JOBS`](../v20.1/show-jobs.html). [#44180][#44180]
+- Previously, CockroachDB could crash in special circumstances when using the vectorized execution engine with the `vectorize=experimental_on` setting. This is now fixed. [#44144][#44144]
+- Fixed planning bug related to FULL joins between single-row relations. [#44156][#44156]
+- Fixed "CopyFrom requires empty destination" internal error. [#44114][#44114]
+- Fix a bug where multiple nodes attempted to populate the results for [`CREATE TABLE ... AS`](../v20.1/create-table-as.html) leading to duplicate rows. [#43840][#43840]
+- All admin users are now allowed to use BACKUP/RESTORE and IMPORT. [#44250][#44250]
```

Comment:
Could link to these docs.  Should monospace SQL statement proper names. (General comment for all SQL statements mentioned on this page).